dialog/file.go

```diff
@@ -97,11 +114,11 @@ func (f *fileDialog) makeUI() fyne.CanvasObject {
 				func(ok bool) {
 					f.win.Hide()
 					if !ok {
-						callback(nil, nil)
+						// callback(nil, nil)
```

Comment:
Is this supposed to be commented out?

Reply:
Probably not. This wasn't really ready for review, I was probably testing something.